### PR TITLE
add section on recommendation of listing shields.io maturity badges

### PR DESCRIPTION
**Pull request by Mordil**
/CC @ktoso

### CODE REVIEW

**Review by ianpartridge — APPROVED** [review]

**Review by ktoso — APPROVED** [review]
LGTM 👍 

// Consensus on the forums was reached that these are quite nice I believe https://forums.swift.org/t/sswg-incubation-process-badges/26660/11?u=ktoso

File: process/incubation.md [hunk 1]
```diff
@@ -47,6 +47,12 @@ Every SSWG project has an associated maturity level: **Sandbox**, **Incubating**
 
 A **supermajority** (two-thirds) is required for a project to be accepted as Incubating or Graduated. If there is not a supermajority of votes to enter at the Graduated level, then the votes toward Graduated are recounted as votes to enter at the Incubating level. If there is not a supermajority of votes to enter at the Incubating level, then all votes are recounted as **sponsorship** to enter at the Sandbox level. If there are not at least two sponsors, the Proposal is rejected.
 
+It is recommended for projects that have been accepted to any of the maturity levels to list the maturity level in their project's README with the appropriate badge as defined:
+
+[![sswg:sandbox|94x20](https://img.shields.io/badge/sswg-sandbox-lightgrey.svg)](https://github.com/swift-server/sswg/blob/master/process/incubation.md#sandbox-level)
+[![sswg:incubating|104x20](https://img.shields.io/badge/sswg-incubating-blue.svg)](https://github.com/swift-server/sswg/blob/master/process/incubation.md#incubating-level)
+[![sswg:graduated|104x20](https://img.shields.io/badge/sswg-graduated-green.svg)](https://github.com/swift-server/sswg/blob/master/process/incubation.md#graduated-level)
+
```
Comment (tomerd) [inline process/incubation.md]: this is fantastic, but i would suggest we move this to the end of the "Ecosystem Index" section

**Comment by tomerd** [comment]
@Mordil thanks for putting together just one comment on placement

File: process/incubation.md (hunk 1 above)
Comment (Mordil) [reply process/incubation.md]: @tomerd Done!

**Comment by kylebrowning** [comment]
👍

**Pull request merged**